cmake/setup.py.cmake

```diff
@@ -27,26 +29,31 @@ if(BUILD_PYTHON)
 
   # Create setup.py
   set(SETUP_PY_IN "${CMAKE_CURRENT_LIST_DIR}/setup.py.in")
-  set(SETUP_PY "${PYTHON_DEST}/setup.py")
-  set(SETUP_PY_INIT "${PYTHON_DEST}/sirf/__init__.py")
-  message(STATUS "setup.py:${SETUP_PY}")
+  set(SETUP_PY "${CMAKE_CURRENT_BINARY_DIR}/cmake/setup.py")
   configure_file("${SETUP_PY_IN}" "${SETUP_PY}")
+  install(FILES "${SETUP_PY}" DESTINATION "${PYTHON_DEST}")
+  message(STATUS "setup.py:${PYTHON_DEST}/setup.py")
+  # create sirf/__init__.py
+  set(PY_MOD_INIT_IN "${CMAKE_CURRENT_LIST_DIR}/sirf.__init__.py.in")
+  set(PY_MOD_INIT "${CMAKE_CURRENT_BINARY_DIR}/cmake/__init__.py")
+  configure_file("${PY_MOD_INIT_IN}" "${PY_MOD_INIT}")
+  install(FILES "${PY_MOD_INIT}" DESTINATION "${PYTHON_DEST}/sirf")
 
   if(PYTHONINTERP_FOUND)
     # python setup.py build
-    add_custom_command(OUTPUT "${SETUP_PY_INIT}"
+    add_custom_command(OUTPUT "${PYTHON_DEST}/sirf/__init__.py"
       COMMAND "${CMAKE_COMMAND}" -E make_directory "${PYTHON_DEST}/sirf"
-      COMMAND "${CMAKE_COMMAND}" -E touch "${SETUP_PY_INIT}"
+      COMMAND "${CMAKE_COMMAND}" -E touch "${PYTHON_DEST}/sirf/__init__.py"
       COMMAND "${PYTHON_EXECUTABLE}" setup.py build
-      DEPENDS "${SETUP_PY}"
+      DEPENDS "${PYTHON_DEST}/setup.py"
```

Comment:
add `DEPENDS ${PYTHON_DEST}/sirf/__init__.py`?

Reply:
ah I see. related to the previous comment... ok

Reply:
that would make the `OUTPUT` not make much sense though

Reply:
think best to just remove the build target altogether